### PR TITLE
fixed the button in the UI which was not consistent

### LINKED ISSUE #3242: UI inconsistent while reporting an error
**Describe the bug**
The buttons in the error report UI is not consistent 

**Expected behavior**
Buttons have equal height

**Steps to reproduce the behavior:**
1. Click Help
2. Send Diagnostic report
-->

**Screenshots**
Screenshot_56

**Environment**
- Version of Kiwix Android : 3.6.0
- Device : Oneplus nord ce
- OS version : SDK33

**Logs**

### PR DESCRIPTION
**Pull request by criticalAY**
Fixes #3242

Screenshot_55

### CODE REVIEW

**Comment by criticalAY** [comment]
@kiwix/lieutenants please could anyone review it

**Comment by criticalAY** [comment]
@kelson42 I am not aware of the reviewing process nor I got response on Slack sorry to ping you but I need little help with the onboarding process

**Comment by 4shutosh** [comment]
Thank you for your efforts, but let's not rush into making contributions. 
Reading the repo readme is a great habit to get started with the project. 
[Here](https://github.com/kiwix/kiwix-android/blob/develop/CONTRIBUTING.md) are the contribution guidelines. 

First of all, the new issues need to be triaged, and if verified and assigned to you, you may proceed to work on them and raise a PR. We do have a format for each and should be followed. Everyone works in different time zones, and you need to be patient regarding the process.

File: core/src/main/res/layout/activity_kiwix_error.xml [hunk 1]
```diff
@@ -25,12 +25,13 @@
 
   <Button
     android:id="@+id/reportButton"
-    android:layout_width="120dp"
+    android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:layout_marginStart="8dp"
     android:layout_marginEnd="8dp"
     android:layout_marginBottom="32dp"
-    android:background="@color/alabaster_white"
```
Comment (gouri-panda) [inline core/src/main/res/layout/activity_kiwix_error.xml:33]: Why did you remove the background color?

**Review by gouri-panda — COMMENTED** [review]
@criticalAY Thanks for the interest in this  project. I have a question regarding the changes you have made. Then we can proceed with the merge with the develop.

File: core/src/main/res/layout/activity_kiwix_error.xml (hunk 1 above)
Comment (criticalAY) [reply core/src/main/res/layout/activity_kiwix_error.xml:33]: The background in this button is not required as it was creating additional padding which made the height of this button not equal with the others

**Comment by criticalAY** [comment]
Is it fine for the test to be running for 2 hours

**Review by MohitMaliFtechiz — APPROVED** [review]
@criticalAY , Thanks. @kelson42 LGTM
